crates/bevy_ecs/src/world/mod.rs

```diff
@@ -320,6 +320,15 @@ impl World {
         Some(EntityRef::new(self, entity, location))
     }
 
+    /// Returns an [`Entity`](crate::entity::Entity) iterator of current entities.
```

Comment:
```suggestion
    /// Returns an [`Entity`] iterator of current entities.
```

This link can be collapsed, as `Entity` is already in scope.

Reply:
Updated in latest change. Thanks!